bokehjs/src/coffee/source/geojson_data_source.coffee

```diff
@@ -0,0 +1,124 @@
+_ = require "underscore"
+{logger} = require "../common/logging"
+ColumnDataSource = require "./column_data_source"
+
+class GeoJSONDataSource extends ColumnDataSource.Model
+
+  initialize: (options) ->
+    @set('geojson', JSON.parse(options.geojson))
+    @set('data', @geojson_to_column_data())
```

Comment:
this is going to send `data` back to the server, for what it's worth. That is hopefully OK.